project_name/settings.py

```diff
@@ -428,3 +431,363 @@
+            'conformity_service': 'notEvaluated',
+            'contact_name': 'Organization Name',
+            'contact_email': 'Email Address',
+            'temp_extent': 'YYYY-MM-DD/YYYY-MM-DD',
+        }
+    }
+}
+
+# If you want to enable Mosaics use the following configuration
+UPLOADER = {
+    'BACKEND': 'geonode.rest',
+    # 'BACKEND': 'geonode.importer',
+    'OPTIONS': {
+        'TIME_ENABLED': True,
+        'MOSAIC_ENABLED': False,
+        'GEOGIG_ENABLED': False,
+    },
+    'SUPPORTED_CRS': [
+        'EPSG:4326',
+        'EPSG:3785',
+        'EPSG:3857',
+        'EPSG:900913',
+        'EPSG:32647',
+        'EPSG:32736'
+    ]
+}
+
+
+### WorldMap Settings ###
+
+
+# for now we remove CsrfViewMiddleware, which creates failures on x-csrftoken
+# We need to fix this 
+MIDDLEWARE_CLASSES = (
+    'corsheaders.middleware.CorsMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.locale.LocaleMiddleware',
+    'pagination.middleware.PaginationMiddleware',
+    # 'django.middleware.csrf.CsrfViewMiddleware', # remove csrf because it will stop users from extranet
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
+    'oauth2_provider.middleware.OAuth2TokenMiddleware',
+    # Add cache middleware for the per-site cache
+    'django.middleware.cache.UpdateCacheMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.cache.FetchFromCacheMiddleware',
+)
+
+INSTALLED_APPS += (
+        'geonode.contrib.worldmap.gazetteer',
+        'geonode.contrib.worldmap.queue',
+        'geonode.contrib.worldmap.wm_extra',
+        'geonode.contrib.createlayer',
+    )
+
+HYPERMAP_REGISTRY_URL = os.getenv('HYPERMAP_REGISTRY_URL', 'http://localhost:8001')
+SOLR_URL = os.getenv('SOLR_URL', 'http://localhost:8983/solr/hypermap/select/')
+MAPPROXY_URL = os.getenv('MAPPROXY_URL', 'http://localhost:8001')
+
+# Other settings
+GEONODE_CLIENT_LOCATION = '/static/worldmap/worldmap_client/'
+USE_GAZETTEER = True
+GAZETTEER_DB_ALIAS = 'wmdata'
+GAZETTEER_FULLTEXTSEARCH = False
+USE_CUSTOM_ORG_AUTHORIZATION = True
+# Modify affiliate defaults
+CUSTOM_ORG_AUTH_TEXT = _("Are you affiliated with Zhejiang University?")
+# Modify the link of terms and condition 
+CUSTOM_AGREE_TOS_TEXT = _("I agree to the <a href='/aboutus/#disclaimer'>Terms and Conditions</a>")
+# Uncomment following line if debugging GeoExplorer static files
+# GEONODE_CLIENT_LOCATION = 'http://localhost:9090/'
+GOOGLE_MAPS_API_KEY = os.getenv('GOOGLE_MAPS_API_KEY')
+
+# Set TIME_ZONE to Shanghai and close USE_TZ to change the time showed in browser
+TIME_ZONE = os.getenv('TIME_ZONE', "Asia/Shanghai")
```

Comment:
this is in GeoNode. If you need to change the settings it is much better to use an env variable.